samples/sample_10_windows/pipeline.yml

```diff
@@ -0,0 +1,16 @@
+workspace:
+  base: c:/gopath/
```

Comment:
It seems like things get messed up if you don't specify this path in Windows.

@bradrydzewski I'm guessing there's some default that could be set here.

Reply:
yes, I will handle that in a separate PR. I believe it is handled by the drone-cli